tests/test_log.c

```diff
@@ -79,9 +79,20 @@ static int __log_pop_failing(
     return -1;
 }
 
+static raft_time_t __raft_clock = 1000000;
```

Comment:
It feels like time just stands still :)

Reply:
This is time in a per-test-process universe, not our universe. :)